winbuild/Makefile

```diff
@@ -136,7 +136,8 @@ TEST_DIR_SRCS_CPP=\
 	tests\ht_test.cpp \
 	tests\ignore_test.cpp \
 	tests\log_stub.cpp \
-	tests\log.cpp
+	tests\log.cpp \
+	tests\string.cpp
 
 OBJS=$(SRCS_CPP:.cpp=.obj) $(SRCS_C:.c=.obj)
 TEST_OBJS=$(TEST_SRCS_CPP:.cpp=.obj) $(TEST_SRCS_C:.c=.obj)
```

Comment:
github won't let me comment on precisely the correct line, but you'll need to add `tests\string.exe` to the list of `TESTS=` that is a couple of lines below this in order for the windows build to build and run your new test